integration-tests/actions/keeper_helpers.go

```diff
@@ -159,6 +160,61 @@ func DeployPerformanceKeeperContracts(
 	return registry, registrar, upkeeps, upkeepIds
 }
 
+// DeployPerformDataCheckerContracts deploys a set amount of keeper perform data checker contracts registered to a single registry
+func DeployPerformDataCheckerContracts(
```

Comment:
I see there are three functions now all doing the same thing except the keeper contract. Time to refactor and take in upkeep type as an argument?